plugins/ceph/check-ceph.rb

```diff
@@ -92,7 +95,7 @@ def run_cmd(cmd)
     pipe, status = nil
     begin
       cmd += config[:cluster] if config[:cluster]
-      cmd += config[:keyring] if config[:keyring]
+      cmd += config[:user] if config[:user]
```

Comment:
Is keyring replaced by user in the command? they don't look the same thing, maybe we can keep keyring, and add user, would it make sense?